SDL/Quintuplet.cu

```diff
@@ -4,6 +4,7 @@
 #include "Quintuplet.cuh"
 #include "allocate.h"
 #include "Kernels.cuh"
+#include "NeuralNetwork.cuh"
```

Comment:
Why was this moved from the header file, Quintuplet.cuh?

Reply:
If it is left in Quintuplet.cuh, we get an error when Quintuplet.cuh is included more than once I think because the Neural Network `runInference` function is now defined in the header.

Reply:
Yeah this was a common issue on the Alpaka branch. Try adding the inline prefix to all of your functions (or at least the ones throwing the linker error) and it should fix it.